### PR TITLE
SAAS:7943: Added some new and open ports plugin

### PR DESCRIPTION
**Pull request by sadeed12345**

### CODE REVIEW

File: plugins/aws/ec2/openLDAP.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        };
+
+        config.ec2_skip_unused_groups = (config.ec2_skip_unused_groups == 'true');
+        
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var ports = {
+            'tcp': [389]
```
Comment (mehakseedat63) [inline plugins/aws/ec2/openLDAP.js:60]: Add UDP

Comment (sadeed12345) [inline plugins/aws/ec2/openLDAP.js:60]:
```suggestion
            'tcp': [389],
            'udp': [389]
```

File: plugins/aws/ec2/openLDAP.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            rcb();
+        }, function(){
+            callback(null, results, source);
+        });
+    },
+    remediate: function(config, cache, settings, resource, callback) {
+        var remediation_file = settings.remediation_file;
+        var putCall = this.actions.remediate;
+        var pluginName = 'openLDAP';
+        var protocol = 'tcp';
```
Comment (sadeed12345) [inline plugins/aws/ec2/openLDAP.js:93]:
```suggestion
        var protocol = ['tcp','udp'];
```

File: plugins/aws/ec2/openCassandraInternode.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Open Cassandra Internode',
+    category: 'EC2',
+    domain: 'Compute',
+    description: 'Determine if TCP port 7001 for Cassandra Internode is open to the public',
```
Comment (sadeed12345) [inline plugins/aws/ec2/openCassandraInternode.js:8]:
```suggestion
    description: 'Determine if TCP port 7000 for Cassandra Internode is open to the public',
```

File: plugins/aws/ec2/openCassandraInternode.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        };
+
+        config.ec2_skip_unused_groups = (config.ec2_skip_unused_groups == 'true');
+        
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var ports = {
+            'tcp': [7001]
```
Comment (sadeed12345) [inline plugins/aws/ec2/openCassandraInternode.js:60]:
```suggestion
            'tcp': [7000]
```

File: plugins/aws/ec2/openCassandraInternode.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        }, function(){
+            callback(null, results, source);
+        });
+    },
+    remediate: function(config, cache, settings, resource, callback) {
+        var remediation_file = settings.remediation_file;
+        var putCall = this.actions.remediate;
+        var pluginName = 'openCassandraInternode';
+        var protocol = 'tcp';
+        var port = 7001;
```
Comment (sadeed12345) [inline plugins/aws/ec2/openCassandraInternode.js:94]:
```suggestion
        var port = 7000;
```

File: plugins/aws/ec2/openLDAP.js [hunk 6]
```diff
@@ -0,0 +1,109 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Open LDAP',
+    category: 'EC2',
+    domain: 'Compute',
+    description: 'Determine if TCP port 389 for LDAP is open to the public',
```
Comment (sadeed12345) [inline plugins/aws/ec2/openLDAP.js:8]:
```suggestion
    description: 'Determine if TCP or UDP port 389 for LDAP is open to the public',
```

File: plugins/aws/ec2/openLDAP.js [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Open LDAP',
+    category: 'EC2',
+    domain: 'Compute',
+    description: 'Determine if TCP or UDP port 389 for LDAP is open to the public',
+    more_info: 'While some ports such as HTTP and HTTPS are required to be open to the public to function properly, more sensitive services such as Kibana should be restricted to known IP addresses.',
+    link: 'http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html',
+    recommended_action: 'Restrict TCP port 389 to known IP addresses',
```
Comment (sadeed12345) [inline plugins/aws/ec2/openLDAP.js:11]:
```suggestion
    recommended_action: 'Restrict TCP or UDP port 389 to known IP addresses',
```

File: exports.js [hunk 8]
```diff
@@ -289,6 +299,7 @@ module.exports = {
         'iamRoleLastUsed'               : require(__dirname + '/plugins/aws/iam/iamRoleLastUsed.js'),
         'maxPasswordAge'                : require(__dirname + '/plugins/aws/iam/maxPasswordAge.js'),
         'minPasswordLength'             : require(__dirname + '/plugins/aws/iam/minPasswordLength.js'),
+        'usersToChangePasswords'        : require(__dirname + '/plugins/aws/iam/usersToChangePasswords.js'),
```
Comment (sadeed12345) [inline exports.js:302]:
```suggestion
```

**Review by AkhtarAmir — APPROVED** [review]

**Pull request merged**